source/mainnet/net/guides/add-delegation.rst

```diff
@@ -0,0 +1,171 @@
+
+                #. In the Desktop Wallet, select **Export transaction proposal**.
+
+                #. Navigate to the location on your computer where you want to save the file. If you're on Windows make sure that **All Files** is selected in **Save as type**. Give the file a name and the extension .json, and then click **Save**.
+
+                #. You have to export the transaction proposal and send it to the co-signer through a secure channel. Optionally, you can also send the Identicon to the co-signers through a different secure channel.
+
+                **Receive signatures from co-signers**
+
+                When the co-signers have signed the transaction, they return the signed transaction proposal to you, and you have to import the files into the Desktop Wallet before you can submit the transaction to the chain.
+
+                #. If you’re still on the same page, go to step 3. If you left the page with the account transaction, go to **Multi-signature Transactions**, and then select Your proposed transactions.
+
+                #. Select the transaction that you want to submit to the chain. You can see an overview of the transaction details and an overview of the signatures. You can also see that the status of the transaction is Unsubmitted, and you can see the identicon and the transaction hash.
+
+                #. Select **Browse to file** and then navigate to the location on your computer where you saved the signed transaction files. Select the relevant files, and then select **OK**. The files are uploaded to the Desktop Wallet and added to the list of signatures. Alternatively, you can drag and drop the signature files from their location on the computer and onto the Desktop Wallet.
+
+                .. _submit-delegation:
+
+                **Submit the transaction to the blockchain**
+
+                When you have received and added all the required signatures, you can submit the transaction to the blockchain.
+
+                #. Review the transaction details carefully to ensure that all information is correct.
+
+                #. Select **I understand this is the final submission and that it cannot be reverted**.
+
+                    If you don’t want to submit the transaction to the chain, you can select **Cancel**. The proposal is no longer active. However, it is still visible in the list of proposals.
+
+                #. Select **Submit transaction to chain**. The transaction is submitted to the chain and finalized on the Ledger.
+
+                #. Select **Finish** to leave the page.
+
+    .. tab:: Mobile Wallet
+
+        #. Go to the **Accounts** screen.
+
+        #. Tap on the balance area of the account you want to delegate from or tap **More** |moredetails|.
+
+        #. Tap the hamburger menu |hamburger| and tap **Delegation**. If you are delegating for the first time, you see some information about delegation.
+
+            .. image:: ../images/mobile-wallet/account-hamburger-menu.png
```

Comment:
The last item in the hamburger menu looks strange with the ellipsis, does it actually look like that? :thinking:

Reply:
Yes.